DataProcessing/datax-host/src/main/scala/datax/executor/ExecutorHelper.scala

```diff
@@ -0,0 +1,75 @@
+// *********************************************************************
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License
+// *********************************************************************
+package datax.executor
+
+import datax.config.ConfigManager
+import datax.constants.{BlobProperties, JobArgument}
+import datax.fs.HadoopClient
+import datax.host.SparkJarLoader
+import datax.securedsetting.KeyVaultClient
+import org.apache.log4j.LogManager
+import org.apache.spark.broadcast
+import org.apache.spark.sql.SparkSession
+
+object ExecutorHelper {
+  private val logger = LogManager.getLogger(this.getClass)
+
+  /***
+    * Create broadcast variable for blob storage account key
+    * @param path blob storage path
+    * @param spark SparkSession
+    */
+  def createBlobStorageKeyBroadcastVariable(path: String, spark : SparkSession): broadcast.Broadcast[String] ={
+    addJarToExecutor(spark)
+    val sc = spark.sparkContext
+    val sa = getStorageAccountName(path)
+    var key = ""
+    KeyVaultClient.withKeyVault {vaultName => key = HadoopClient.resolveStorageAccount(vaultName, sa).get}
+    val blobStorageKey = sc.broadcast(key)
```

Comment:
Is blobStorageKey being used anywhere?

Reply:
Yes. It is being returned by this function. No return keyword required in scala :)